packages/dds/map/src/test/memory/directory.spec.ts

```diff
@@ -57,30 +33,49 @@ describe("SharedDirectory memory usage", () => {
         // See the comment at the top of the test suite for more details.
     });
 
-    benchmarkMemory({
-        title: "Create empty directory",
-        minSampleCount: 1000,
-        benchmarkFn: async () => {
-            createLocalDirectory("testDirectory");
-        },
-    });
+    benchmarkMemory(new class implements IMemoryTestObject {
+        title = "Create empty directory";
```

Comment:
Let's give this one a `minSampleCount = 500`, to keep with what we were doing before. It should be a pretty lightweight one, so more iterations shouldn't require much more clock time, and it should help us get more accurate measurements.